README.md

```diff
@@ -104,11 +104,12 @@ These need to be further massaged in MongoDB.
         |------|------|
         |resourceProvider-Xx.json|-c resourceProvider|
         |health-Xx.json|-c health|
-        |role-Xx.json|-c role|
-        |policy-Xx.json|-c policy|
-        |portalextension-Xx.json|-c portal**E**xtension|
-        
+        |roles-Xx.json|-c role|
+        |policies-Xx.json|-c policy|
+        |portalextensions-Xx.json|-c portal**E**xtension|
+            
     * Files need to be loaded in the following order: Ww, Ff, Mc, Bf. Note that all the files for a given type, irrespective of the cloud, are loaded into the same collection.
+    * When importing **portalextensions**, add ```--jsonArray ``` at end of import command. 
```

Comment:
This shouldn't be necessary. The content of the portal extensions should be all contained within a `value` property like so:

```json
{
  "value" : [
    
    {
      "name": "HubsExtension",
      "validEndpoint": false,
      "regexMatch": false
    },
    {
      "name": "Microsoft_AAD_B2CAdmin",
      "features": {},
      "validEndpoint": true,
      "regexMatch": true
    },
    ...
  ]
}
```

See [Program.cs#L262](https://github.com/gsacavdm/azure-parity/blob/master/Program.cs#L262)